lbrynet/core/HashBlob.py

```diff
@@ -206,6 +214,14 @@ def __init__(self, blob_dir, *args):
         self.file_path = os.path.join(blob_dir, self.blob_hash)
         self.setting_verified_blob_lock = threading.Lock()
         self.moved_verified_blob = False
+        if os.path.isfile(self.file_path):
+            self.set_length(os.path.getsize(self.file_path))
+            # This assumes that the hash of the blob has already been
+            # checked as part of the blob creation process. It might
+            # be worth having a function that checks the actual hash;
+            # its probably too expensive to have that check be part of
+            # this call.
```

Comment:
on startup, if there are a lot of blobs, we'd end up having to read all of them from disk and calculate the hash, which can be quite slow.  Given that the hash was checked when the file was written, I don't think its necessary to recheck each time a HashBlob is initialized.

Also, its more code.  I'm attempting to justify not making the check.